examples/05_task_input_data.py

```diff
@@ -91,13 +91,16 @@
             # Here we don't use dict initialization.
             td = rp.TaskDescription()
             td.executable     = '/usr/bin/wc'
-            td.arguments      = ['-c', 'input.dat']
+            td.arguments      = ['-c', 'input_1.dat', 'input_2.dat']
           # td.input_staging  = ['input.dat']
 
           # this is a shortcut for:
-            td.input_staging  = {'source': 'client:///input.dat',
-                                 'target': 'task:///input.dat',
-                                 'action': rp.TRANSFER}
+            td.input_staging  = [{'source': 'client:///input.dat',
+                                  'target': 'task:///input_1.dat',
+                                  'action': rp.TRANSFER},
+                                 {'source': 'https://1.1.1.1/',
```

Comment:
For this line: `source': 'https://1.1.1.1/` is it missing the `input_2.dat` in the end so it can be like:
`https://1.1.1.1/input_2.dat`? Or are we expecting only the remote path without the file name in `source` and then concatenating `source` + `target` when we download or something like that?

Also, for `https://1.1.1.1/input_2.dat`, when I try to fetch the file I get a file not found from the server!

Reply:
Got it and make sense, thanks.